examples/tooltip.py

```diff
@@ -0,0 +1,131 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+import dash_mantine_components as dmc
+
+
+pio.renderers.default = 'browser'
```

Comment:
What's this for?

Reply:
Not necessary here. Just removed.

I use it for local development when I have interactive session.